content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md

```diff
@@ -1,18 +1,18 @@
 ---
 reviewers:
 - sig-cluster-lifecycle
-title: Set up a Highly Availabile etcd Cluster With kubeadm
+title: Set up a High Availability etcd cluster with kubeadm
```

Comment:
Why is High Availability all-capsed? Do we have style guidance?

Reply:
No caps, adverb: _highly available_. See [this example](https://kubernetes.io/docs/setup/independent/high-availability/) from our own docs.